Source/dom/export.js

```diff
@@ -46,58 +46,146 @@ export const DEFAULT_EXPORT_OPTIONS = {
  *
  *
  * @param {dictionary} options Options indicating which sizes and formats to use, etc.
+ *
+ * @returns If an output path is not set, the data is returned
  */
 export function exportObject(object, options) {
-  const merged = { ...DEFAULT_EXPORT_OPTIONS, ...options }
+  if (!object) {
+    throw new Error('No object provided to export')
+  }
+  let exportJSON = false
+  let { formats } = options
+
+  if (typeof formats === 'string') {
+    formats = formats
+      .split(',')
+      .map(format => format.trim())
+      .filter(t => t.length > 0)
+  }
+  if (formats) {
+    const idx = formats.indexOf('json')
+    if (idx >= 0) {
+      formats.splice(idx, 1)
+      exportJSON = true
+    }
+    formats = formats.join(',')
+  }
+
+  const merged = {
+    ...DEFAULT_EXPORT_OPTIONS,
+    ...options,
+    formats,
+  }
+  const objectsToExport = (Array.isArray(object) ? object : [object]).map(
+    o => (isWrappedObject(o) ? o.sketchObject : o)
+  )
+  if (!objectsToExport) {
```

Comment:
It will always be truthy here

```suggestion
  if (!objectsToExport.length) {
```